components/ide/code/leeway.Dockerfile

```diff
@@ -46,6 +47,8 @@ RUN yarn --frozen-lockfile --network-timeout 180000
 RUN rm -rf remote/node_modules/
 COPY --from=dependencies_builder /gp-code/remote/node_modules/ /gp-code/remote/node_modules/
 
+# todo(ft): add check for whether codeCommit matches codeVersion
```

Comment:
This should be implemented before merging to ensure version consistency.

Reply:
Done, should always check now :)